File: algorithms/100158.is_unique_lcci/is_unique_lcci.rs
impl Solution {
    pub fn is_unique(astr: String) -> bool {
        let mut chars = [false; 26];
        for &s in astr.as_bytes().iter() {
            let c = s as usize - 97;
            if chars[c] {
                return false;
            } else {
                chars[c] = true;
            }
        }
        true
    }
}

File: algorithms/100158.is_unique_lcci/is_unique_lcci.cpp
class Solution {
public:
    bool isUnique(const string& astr) {
        int64_t bits = 0;
        for (const char&c: astr) {
            if ((bits & (1 << (c - 'a'))) != 0) {
                return false;
            }
            bits |= (1 << (c - 'a'));
        }
        return true;
    }
};